riscv-asm.md

```diff
@@ -183,6 +183,41 @@ Directive    | Arguments                      | Description
 .balign      | b,[pad_val=0]                  | byte align
 .zero        | integer                        | zero bytes
 .variant_cc  | symbol_name                    | annotate the symbol with variant calling convention
+.attribute   | name, value                    | RISC-V object attributes, more detailed description see [.attribute](#.attribute).
+
+## <a name=.attribute></a> `.attribute`
+
+`.attribute` directive is used for record information about an object
+file/binary that a linker or runtime loader needs to check compatibility.
+
+More information can refer [attribute section in RISC-V psABI](https://github.com/riscv-non-isa/riscv-elf-psabi-doc/blob/master/riscv-elf.adoc#attributes).
+
+`.attribute` take two argument, first argument of `.attribute` is the symbolic
+name of attribute or the attribute number, the prefix `Tag_RISCV_` can be
+omitted, and second argument is string if attribute number is odd and an integer
```

Comment:
Why "can be omitted"? Why not just mandate that it's used without the `Tag_RISCV_` prefix? I'm not seeing the value in that flexibility, and if there's no reason we might as well just keep the implementation simpler.

Reply:
I would prefer to keep that for compatibility since we already ship that on both GNU toolchain  and LLVM toolchain for a while.

Reply:
And I must admit that seems redundant...